pages/common/perl.md

```diff
@@ -21,3 +21,11 @@
 - Run a Perl script in debug mode, using `perldebug`:
 
 `perl -d {{script.pl}}`
+
+- Replace all occurrences of a string in a file with another string, overwriting the file in-place:
+
+`perl -p -i -e 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
`-e` refers to an expression (rather than a perl script), and is explained in the previous examples on this page (which refer to it as a "statement" -- perhaps we should tweak those to say "expression", to improve the memorability).

`-i` is the option that stands for in-place, which is easy enough to describe.

`-p`, as described in #1239, seems to stand for "program", but is not really something that can be explained succinctly -- I'd rather leave it unexplained, unless you guys can come up with something nice :)